### PR TITLE
Fix workspace Integration Tests

### PR DESCRIPTION
**Pull request by Alfred-Mountfield**
## 🌟 What is the purpose of this PR?

Fairly self-explanatory, also deleted outdated tests made redundant by #1454 

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1201893074552404/1203391635298904/f) _(internal)_

## 🚫 Blocked by

- [x] #1457 
- [x] #1458 
- [x] #1460 

## 🔍 What does this change?

See commit list

## 🛡 What tests cover this?

These are tests, so all jest tests inside the integration folder

## ❓ How to test this?

1.  See CI, or,
1.  Run `yarn workspace @hashintel/hash-integration jest`

## 📹 Demo

image

### CODE REVIEW

**Review by benwerner01 — APPROVED** [review]
Great work, happy to see the tests passing again :)

File: packages/hash/api/src/model/knowledge/block.model.ts [hunk 1]
```diff
@@ -33,7 +33,10 @@ export default class extends EntityModel {
       );
     }
 
-    return new BlockModel({ entity, entityTypeModel: entity.entityTypeModel });
+    return new BlockModel({
+      entity: entity.entity,
+      entityTypeModel: entity.entityTypeModel,
+    });
```
[the remark this answers is not in the document]
Comment (Alfred-Mountfield) [reply packages/hash/api/src/model/knowledge/block.model.ts:39]: I was just fixing what was there 😁 This has been fixed in the follow-up PR anyway

File: packages/hash/integration/src/tests/util.ts [hunk 2]
```diff
@@ -1,68 +1,10 @@
...
-  deprecatedUpdateEntityType,
-  getEntityAndLinks,
-} from "../graphql/queries/entity.queries";
-import {
-  createPage,
-  getAccountPagesTree,
-  getPage,
-  setPageParent,
-  updatePageContents,
-} from "../graphql/queries/page.queries";
```
[the remark this answers is not in the document]
Comment (Alfred-Mountfield) [reply packages/hash/integration/src/tests/util.ts:65]: I've deleted them, which also meant we didn't need GraphQL at all here. We can always re-add them if we want more integration tests but I think the intention is to rely on playwright more

[the remark this answers is not in the document]
Comment (Alfred-Mountfield) [reply packages/hash/integration/src/tests/util.ts:65]: Actually hold on, I've introduced a bug, we needed a scalar for OrgSize 🙃

[the remark this answers is not in the document]
Comment (Alfred-Mountfield) [reply packages/hash/integration/src/tests/util.ts:65]: Alright should be good now
image

**Review by benwerner01 — APPROVED** [review]

**Pull request merged**